### PR TITLE
Adding a new error

### PR DESCRIPTION
**Pull request by joinimran**
As reported, the new error information has been added in the document.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6399

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/known-issues.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ ms.topic: article
 <tr><td>Blocking apps specified in a user-targeted Enrollment Status Profile are ignored during device ESP.</td>
 <td>The services responsible for determining the list of apps that should be blocking during device ESP are not able to determine the correct ESP profile containing the list of apps because they do not know the user identity.  As a workaround, enable the default ESP profile (which targets all users and devices) and place the blocking app list there.  In the future, it will be possible to instead target the ESP profile to device groups to avoid this issue.</tr>
 
+<tr><td>That username looks like it belongs to another organization. Try signing in again or start over with a different account.</td>
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/known-issues.md:32]:
```suggestion
<tr><td>The username looks like it belongs to another organization. Try signing in again or start over with a different account.</td>
```

File: windows/deployment/windows-autopilot/known-issues.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ ms.topic: article
...
+  <td>Confirm that all of your information is correct at HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Provisioning\Diagnostics\AutoPilot see <a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/troubleshooting#windows-10-version-1709-and-above">Troubleshooting Windows Auto Pilot </a> for more details.</td></tr>
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/known-issues.md:33]:
```suggestion
  <td>Confirm that all of your information is correct at HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Provisioning\Diagnostics\AutoPilot. See [Troubleshooting Windows Autopilot](https://docs.microsoft.com/windows/deployment/windows-autopilot/troubleshooting#windows-10-version-1709-and-above) for more details.</td></tr>
```

Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/known-issues.md:33]:
```suggestion
  <td>Confirm that all of your information is correct at HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Provisioning\Diagnostics\AutoPilot. See <a href="https://docs.microsoft.com/windows/deployment/windows-autopilot/troubleshooting#windows-10-version-1709-and-above">Troubleshooting Windows Auto Pilot </a> for more details.</td></tr>
```

**Comment by joinimran** [comment]
Suggested changes applied.

**Pull request merged**